recipes-ccsp/ccsp/rdk-wanmanager.bb

```diff
@@ -8,7 +8,7 @@ DEPENDS_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'rdkb_wan_manager', '
 
 require recipes-ccsp/ccsp/ccsp_common.inc
 
-GIT_TAG = "v2.3.0"
+GIT_TAG = "RC2.4.0a"
```

Comment:
I wonder should a release candidate change goes to a feature / sprint branch instead?